gems/jquery-rails/CVE-2015-1840.yml

```diff
@@ -1,11 +1,9 @@
 ---
 gem: jquery-rails
 cve: 2015-1840
-url: https://groups.google.com/forum/#!topic/ruby-security-ann/XIZPbobuwaY
-title: |
-  CSRF Vulnerability in jquery-ujs and jquery-rails
+url: http://blog.honeybadger.io/understanding-the-rails-jquery-csrf-vulnerability-cve-2015-1840/
```

Comment:
OK, so - url I feel should refer to canonical upstream disclosure.

I definitely like the idea of linking explanatory posts, though. Can you think of an acceptable schema addition?